### PR TITLE
KM-2: Create user entity

### PR DESCRIPTION
**Pull request by jobayerccj**

### CODE REVIEW

File: src/Entity/User.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\Column(length: 255)]
+    private ?string $name = null;
+
+    #[ORM\Column(length: 255)]
```
Comment (ronykader) [inline src/Entity/User.php:23]: We need to implement validation, such as ensuring the database field is unique, and we also need to confirm its validation with the entity.

File: src/Entity/User.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    {
+        return $this->status;
+    }
+
+    public function setStatus(?bool $status): static
+    {
+        $this->status = $status;
+
+        return $this;
+    }
```
Comment (ronykader) [inline src/Entity/User.php:113]: We need to include two additional fields: 'created_at' and 'updated_at'.

**Review by ronykader — CHANGES_REQUESTED** [review]
1. We need to implement validation, such as ensuring the database field is unique, and we also need to confirm its validation with the entity.
2. We need to include two additional fields: 'created_at' and 'updated_at'.

File: src/Entity/User.php (hunk 2 above)
Comment (jobayerccj) [reply src/Entity/User.php:113]: we are using TimestampableEntity which will automatically insert those columns. Please check migration file.

File: src/Entity/User.php (hunk 1 above)
Comment (jobayerccj) [reply src/Entity/User.php:23]: We will add those conditions when we will complete form submission for those columns

**Pull request merged**